test/e2e/runner/rpc.go

```diff
@@ -13,7 +13,7 @@ import (
 )
 
 const (
-	WaitForHeightTimeout = 20 * time.Second
+	WaitForHeightTimeout = time.Minute
```

Comment:
is this still needed, or just for debugging?

Reply:
timeout was increased by `tendermint` team ([here](https://github.com/tendermint/tendermint/blob/master/test/e2e/runner/rpc.go#L112)), without this (locally), tests always failed